specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2025-06-01/mfe.json

```diff
@@ -13110,6 +13318,142 @@
       },
       "additionalProperties": false
     },
+    "CapabilityHost": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceBase"
+        }
+      ],
```

Comment:
>items [](http://example.com/codeflow?start=11&length=5)

Are these Urls? If so, add format: uri. If these are resourceIds, then you should add format: arm-id + x-ms-arm-id-details as appropriate. Depending on what they are, you might also find a better name for the property. In any case, you should beef up the description to explain how connections are represented by these items.